src/packages/console/src/pages/createSitePage.tsx

```diff
@@ -143,26 +139,15 @@ export const CreateSitePage = (): React.ReactElement => {
               ) : (
                 <Form isLoading={isLoading || accounts === undefined} onFormSubmitted={onCreateSiteClicked}>
                   <Stack direction={Direction.Vertical} shouldAddGutters={true} isFullWidth={true} defaultGutter={PaddingSize.Wide}>
-                    <FormControl
-                      variant='outlined'
-                      margin='normal'
-                      required
-                      fullWidth
-                    >
-                      <InputLabel id='account-select-label'>Account</InputLabel>
-                      <Select
-                        labelId='account-select-label'
-                        value={selectedAccountId}
-                        onChange={onAccountSelected}
-                        error={selectedAccountIdError !== undefined}
-                        label='Account'
-                        // helperText={selectedAccountIdError}
-                      >
-                        {accounts?.map((account: Account, index: number): React.ReactElement => (
-                          <MenuItem key={index} value={account.accountId}>{account.name}</MenuItem>
-                        ))}
-                      </Select>
-                    </FormControl>
+                    <OptionSelect
+                      selectedItemKey={String(selectedAccountId)}
+                      inputWrapperVariant={selectedAccountIdError && 'error'}
+                      messageText={selectedAccountIdError} onItemClicked={onAccountSelected}
+                      options={accounts?.map((account: Account) => ({ itemKey: String(account.accountId),
```

Comment:
rendering should go on a newline as well like:

```
prop={......: React.ReactElement => (
  <Component .... />
)}
```

Reply:
ah this isnt rendering, wierd. will think of a better format just a sec.

Reply:
ah yes use the same format pls (and add typing)

Reply:
didn't get you

Reply:
Do you mean the itemKey needs to be on newline ?

Reply:
itemKey and the bracket just before it (not the round bracket, just the curly one)